proto/panacea/datapool/v2/tx.proto

```diff
@@ -43,6 +46,16 @@ message MsgRegisterDataValidator {
 message MsgRegisterDataValidatorResponse {
 }
 
+// MsgUpdateDataValidator defines the Msg/UpdateDataValidator request type.
+message MsgUpdateDataValidator {
+  DataValidator validator_detail = 1;
```

Comment:
It'd be better to specify which fields can be updated, as Cosmos does: https://github.com/cosmos/cosmos-sdk/blob/6f79bf341741ab1f6c68804f4f113753d70179fe/proto/cosmos/staking/v1beta1/tx.proto#L65

Reply:
Thanks, I'll change the comment.

Reply:
Ah. I meant defining updatable fields explicitly in the `MsgUpdateDataValidator` message, instead of using the `DataValidator` message as it is:
```proto
message MsgUpdateDataValidator {
  string endpoint = 1;
}
```
or so

Reply:
Ah Okay I understand. The `--from` flag can define the `datavalidator` address which wants to update. I'll change it.

Reply:
Ah. 1 sec.. For the `--from` flag, we may need the `sender` field or so. Let me check.

Reply:
Yeah. You need the `validator_address` in the `MsgUpdateDataValidator` for verifying the tx signature.

Reply:
Then, let's go with 
```
message MsgUpdateDataValidator {
  string data_validator_address = 1; // signer
  string endpoint = 2;
}
```

Even though it looks a duplicated definition with the `DataValidator` message, it may be safer for the future.

Reply:
Yes, but the `validator_address` and `endpoint` were included in `DataValidator` type. So l thought that there have no changes or rename the field name(`validator_detail`) to `update_data_validator`.

Reply:
Okay, I'll change it.